libgnucash/engine/Account.cpp

```diff
@@ -113,6 +113,7 @@ enum
 
     PROP_LOT_NEXT_ID,                   /* KVP */
     PROP_ONLINE_ACCOUNT,                /* KVP */
+    PROP_OPENING_BALANCE,               /* KVP */
```

Comment:
Name this PROP_IS_OPENING_BALANCE to avoid confusion with the value of the account's opening balance.